internal/twirptest/service_test.go

```diff
@@ -1200,3 +1201,40 @@ func TestContextValues(t *testing.T) {
 		t.Errorf("Client err=%q", err)
 	}
 }
+
+func TestPanicFlushing(t *testing.T) {
+	h := PanickyHatmaker("bang!")
+	s := httptest.NewUnstartedServer(NewHaberdasherServer(h, nil))
+	defer s.Close()
+	s.Config.ErrorLog = testLogger(t)
```

Comment:
Is not clear what is being tested or setup here. Maybe this line needs a comment explaining what to expect?

Reply:
You could change the input to `func(args ...interface{})` and pass in `t.Log` to be more explicit about how `t` can be used.